### ISSUE TITLE
focus indicator not visible for the new notebook cell menu items

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#10712]
the menu items: x, + and delete do not have focus indicator visible.
<img width="334" alt="Screen Shot 2020-06-03 at 9 26 56 AM" src="https://user-images.githubusercontent.com/13777222/83692486-f747fb80-a5a8-11ea-84e5-ac801e8adbc6.png">

**Comment by chlafreniere** [comment]
@halerankin can you look at this next? Is there something pretty easy that we can do for this?
